### DISCUSSION TITLE
Задание 2

### DISCUSSION THREAD

**Discussion by AmastyVadim** [daniil-loban/AmastyTraining#7] [General]
Оценка: 9

Замечания смотрите тут: https://github.com/daniil-loban/AmastyTraining/pull/4#pullrequestreview-724185228
И есть ещё одно общее: отступы в xml и phtml файлах делайте, пожалуйста, в 4 пробела. Это не js)) (Хотя и в js в Magento принято делать отступы в 4 пробела)

**Comment by daniil-loban** [comment]
спасибо, про отступы понял, про замечания - результат генераций плагинами стоит ли удалять это ?

**Reply by AmastyVadim** [reply]
Не совсем понял про какой результат идёт речь)

**Reply by daniil-loban** [reply]
переопределение конструктора и phpDoc для него

**Reply by AmastyVadim** [reply]
Стоит ли удалять phpDoc и переопределение конструктора? Если вопрос про это, то да)
